src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly2.java

```diff
@@ -25,29 +30,29 @@ public class InputLeftCurlyCommentBeforeLeftCurly2 {
     private long[] countList;
 
     void method1() /* comment
-    comment */ { // ok until #11410
+    comment */ { // violation ''{' at column 16 should be on the previous line'
     }
 
     void method2() /********
-    comment */ {  // ok
+    comment */ { // violation ''{' at column 16 should be on the previous line'
         if (!Arrays.equals(this.countList, countList)
-            /* || !Arrays.equals(this.whereCreated, other.whereCreated) */ ) { // ok
+            /* || !Arrays.equals(this.whereCreated, other.whereCreated) */ ) {
         }
     }
 
     void method3() /*****
-    ********/ { } // ok until #11410
+    ********/ { } // violation ''{' at column 15 should be on the previous line'
 
 
-    InputLeftCurlyCommentBeforeLeftCurly2() /**************/ { } // ok
+    InputLeftCurlyCommentBeforeLeftCurly2() /**************/ { }
 
     InputLeftCurlyCommentBeforeLeftCurly2(int data) /*
-     ****** comment *********/ { } // ok until #11410
+     ****** comment *********/ { } // violation ''{' at column 32 should be on the previous line'
 }
 
 class Class {
     private
-    class Node /************/{
+    class Node /************/ {
```

Comment:
should be ok to keep no space, please keep no-space, we need test code like this.

Reply:
done